src/Paket.Core/InstallModel.fs

```diff
@@ -400,7 +438,7 @@ type InstallModel with
 
     member this.AddTargetsFiles targetsFiles = InstallModel.addTargetsFiles targetsFiles this
 
-    member this.AddPackageFile (path, file, references) = InstallModel.addPackageFile path file references this
+    //member this.AddPackageFile (path, file, references) = InstallModel.addPackageFile path file references this
```

Comment:
Strictly speaking this was a public API, but it wasn't used anywhere and I'd like to get rid of it because it doesn't consider the `new` "ref" folder.